www/src/Locales/en/AddonsConfig.jsx

```diff
@@ -56,6 +56,19 @@ export default {
 	'dual-directional-input-dpad-mode-label': 'Dual D-Pad Mode',
 	'dual-directional-input-combine-mode-label': 'Combination Mode',
 	'dual-directional-input-four-way-joystick-mode-label': 'Dual Directional 4-Way Joystick Mode',
+	'AddonsConfig:tilt-header-text': 'Tilt Input',
```

Comment:
You don't need to prefix with `AddonsConfig:` here. Those prefixes are used in the `t` translate function to specify the namespace, namespace being the AddonsConfig file.